### DISCUSSION TITLE
OPC Sever in FUXA

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#559] [Ideas]
Hi, 
There are many cases where data from SCADA applications such as **Fuxa** are required to be monitored in other SCADA monitoring systems. I believe that an **OPC server** would be welcome for such cases. An **OPC server** would thus be a bridge between **Fuxa** and other SCADA applications. An example would be a Central Server where the data from the **FUXA** application are collected through an **OPC server**. I wonder how easy it is to integrate such an **OPC server** in **Fuxa**?

**Comment by IonuDragomir1** [comment]
I agree that an OPC server would be very useful for FUXA in order to ease the data transfer towards different other applications, like SCADA, etc. From my experience, there are many situation when OPC server is needed.
